src/main/java/me/hydos/rosella/render/renderer/Renderer.java

```diff
@@ -228,7 +229,7 @@ public void freeSwapChain() {
         }
 
         vkDestroyRenderPass(rosella.common.device.getRawDevice(), mainRenderPass.getRawRenderPass(), null);
-        swapchain.getSwapChainImageViews().forEach(imageView ->
+        swapchain.getSwapChainImageViews().forEach((LongConsumer) imageView ->
```

Comment:
why

Reply:
:conc:

Reply:
Ambiguous with forEach(Consumer<Long>) and forEach(LongConsumer)